galasa-ui/src/app/layout.tsx

```diff
@@ -7,10 +7,13 @@ import PageHeader from '@/components/PageHeader';
 import '../styles/global.scss';
 
 export default function RootLayout({ children }: { children: React.ReactNode }) {
+
+  let galasaServiceName = process.env.NEXT_PUBLIC_GALASA_SERVICE_NAME?.trim() || "Galasa Service";
```

Comment:
I'd set this as a `const` since it shouldn't change - also see the comment below about passing this as a property into the PageHeader component to avoid duplication

One more thing - it would be good to add `NEXT_PUBLIC_GALASA_SERVICE_NAME` to the .env file so that other developers know that this env variable is being used